.env.example

```
@@ -0,0 +1,6 @@
+COUCHDB_URL=http://admin:adminpass@localhost:5984
+COUCHDB_DB_ITEMS=items
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The COUCHDB_DB_ITEMS key should go before the COUCHDB_URL key

(UnorderedKey)